src/lifx/utils.js

```diff
@@ -125,20 +186,22 @@ utils.rgbHexStringToObject = function(rgbHexString) {
   };
 };
 
+/**
+ * find mininum number in an array
+ * @param {Array} array of numbers
+ * @return {Number} minium of the given array
+ */
 utils.minNumberInArray = function(array) {
-  const sortedCopy = array.slice();
-  sortedCopy.sort(function(a, b) {
-    return a - b;
-  });
-  return sortedCopy[0];
+  return Math.min.apply(undefined, array);
```

Comment:
This is a nice cleanup but if the current babel settings allow, use spread instead of `apply`? It's not that important though.

Reply:
mist remerge with the on of the prev PR.
fixed